server/src-lib/Hasura/RQL/DML/Select/Types.hs

```diff
@@ -225,3 +227,16 @@ mergeArrNodes lNode rNode =
   where
     ArrNode lExtrs colMapping lBN = lNode
     ArrNode rExtrs _          rBN = rNode
+
+getJSONArgumentPath :: AnnArgs -> Maybe T.Text
```

Comment:
We may not require this function if we use `ColOp` kind of data type.

Reply:
Is this function relevant?

Reply:
Sorry, I forgot